tests/testthat/test-write_eml.R

```diff
@@ -22,6 +22,17 @@ test_that("write_eml() returns the expected eml.xml file for the example
            dataset", {
   skip_if_offline()
   x <- example_dataset()
+
+  # Adapt contributors to test for bug #166
+  contributors(x) <- contributors(x) %>%
+    tibble::add_row(
+      title = "Sanne Govaert",
```

Comment:
Can you update this to a dummy name, with orcid 0000-0000-0000-0000. That makes it easier to recognize in the snapshot.